molgenis/molgenis-emx2
# fix(directory): quality checkmarks and qualities visible in default s…

[demo-data.zip](https://github.com/molgenis/molgenis-emx2/files/14093641/demo-data.zip)
…ettings

how to test:
- In the preview => create database from biobank_directory template WITHOUT loading example data
- Upload the attached demo-data (this is the current demo-data without the molgenis-settings.csv file)
- Add "directory" menu item (settings => menu => add directory app)
- See the green quality checkmarks for both biobank1 and biobank2 and the quality information on biobank and collection detail cards.

To view that's not working with the current default settings => do the same steps on https://bbmri-emx2-test.molgeniscloud.org

todo:
- [na] updated docs in case of new feature
- [na] added tests
